Kajabi/sage-lib
# Update Sage icon font

## Description
This PR makes changes to the Sage icon font:

### Added:
- <img width="109" alt="icon-ban" src="https://user-images.githubusercontent.com/7331038/102394066-366af680-3fa7-11eb-9661-e57b7427bf53.png">
- <img width="111" alt="icon-lab" src="https://user-images.githubusercontent.com/7331038/102394129-471b6c80-3fa7-11eb-8555-10c48bb2dcc3.png">
- <img width="128" alt="icon-success" src="https://user-images.githubusercontent.com/7331038/102394145-4d114d80-3fa7-11eb-926f-1a35fd2ff077.png">
- <img width="164" alt="icon-super-admin" src="https://user-images.githubusercontent.com/7331038/102394156-51d60180-3fa7-11eb-97b3-ac96bf027eed.png">

### Removed:
- <img width="155" alt="icon-add-image" src="https://user-images.githubusercontent.com/7331038/102394184-5f8b8700-3fa7-11eb-8b64-c2be09709ad6.png">
- <img width="182" alt="icon-folder-group" src="https://user-images.githubusercontent.com/7331038/102394201-64e8d180-3fa7-11eb-9366-cd011ce6f9da.png">
- <img width="117" alt="icon-sort" src="https://user-images.githubusercontent.com/7331038/102394215-6914ef00-3fa7-11eb-9b18-47363edcda62.png">

## Test notes

### Steps for testing
1. Visit `/pages/design/icon` in the Sage Design System
2. Ensure all icons match their name
